Source/WebInspectorUI/UserInterface/Views/FontVariationDetailsSectionRow.js

```diff
@@ -142,6 +165,87 @@ WI.FontVariationDetailsSectionRow = class FontVariationDetailsSectionRow extends
 
         return 1;
     }
+
+    _showValidity()
+    {
+        if (!this._hasValidValue)
+            this.warningMessage = WI.UIString("Axis value outside of supported range: %s – %s", "A warning that is shown in the Font Details Sidebar when the value for a variation axis is outside of the supported range of values").format(this._formatAxisValueAsString(this._minimumValue), this._formatAxisValueAsString(this._maximumValue));
+        else
+            this.warningMessage = null;
+    }
+
+    _handleValueBlurEvent(event)
+    {
+        this._skipUpdatingInputValue = false;
+        this.value = this._value;
+    }
+
+    _handleValueInputEvent(event)
+    {
+        if (this._showValidityTimeout)
+            clearTimeout(this._showValidityTimeout);
+
+        // Allow the input to temporarily hold an invalid value without immediately warning the user.
+        this._showValidityTimeout = setTimeout(this._showValidity.bind(this), 750);
```

Comment:
I think this is going to feel like poor performance from a user perspective. I'd personally prefer instant feedback as a type as to the validity of what I've typed, particularly since we start with values in the field.

Reply:
Ok, I aded another check and removed this timeout altogether.

The main case I wanted to avoid is nagging the user while they:
- deleted the contents of the input in expectation of typing a new value
- just started typing a negative value

Now, we skip showing the warning in those cases. We will immediately show the warning for an otherwise invalid value.